data/src/main/java/me/androidbox/data/mapper/imp/DataToDomainMapperImp.kt

```diff
@@ -0,0 +1,27 @@
+package me.androidbox.data.mapper.imp
+
+import me.androidbox.data.local.entity.EventEntity
+import me.androidbox.data.mapper.DataToDomainMapper
+import me.androidbox.domain.authentication.model.Event
+import javax.inject.Inject
+
+class DataToDomainMapperImp @Inject constructor()
+    : DataToDomainMapper<@JvmSuppressWildcards List<EventEntity>, @JvmSuppressWildcards List<Event>> {
+
+    override fun invoke(listOfEventEntity: List<EventEntity>): List<Event> {
```

Comment:
Apart from my comment above that this is unnecessary, here you'd force the mapper to be called with a list of events. Sometimes, you just have a single event though, for example when going to the detail screen. Then this wouldn't work unless you pass a list with one item which doesn't seem very nice